nntrainer/graph/network_graph.h

```diff
@@ -147,14 +148,15 @@ class NetworkGraph {
   void setBatchSize(unsigned int batch_size);
 
   /**
-   * @brief try apply gradient at the last of gradient access
+   * @brief try apply gradient if possible
    * @note if it is not the last of the gradient access, this is noop
+   * @note if the gradient is to be clipped by norm, this is noop
    *
    * @param node node to try apply gradient
    * @param apply_func apply function
    */
-  void applyGradientsOnLastAccess(LayerNode *node,
-                                  std::function<void(Weight &)> apply_func);
+  static void applyGradients(LayerNode *node,
+                             std::function<void(Weight &)> apply_func);
```

Comment:
nitpicking: can we change apply_func to reference?

Reply:
updated.